magento/pwa-studio
# Pins dependency for npm-run to fix security vulnerability in sync-exec

## Description

Pins our transitive dependency on `npm-run` to a version that does not depend on `sync-exec@0.6.2`, which [has a security vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2017-16024).

## Related Issue





Closes #1656 .

## Verification Steps

1. `yarn clean:all`
1. `yarn install`
1. Verify that `yarn list --pattern sync-exec` does not return any matches
1. Verify that our query validation plugin still works by running `yarn run validate-queries` and ensuring it is successful
1. Verify that `yarn build` succeeds

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
